include/bluetooth/direction.h

```diff
@@ -217,4 +230,21 @@ int bt_df_conn_cte_rx_enable(struct bt_conn *conn, const struct bt_df_conn_cte_r
  */
 int bt_df_conn_cte_rx_disable(struct bt_conn *conn);
 
+/**
+ * @brief Set Constant Tone Extension transmission parameters for a connection.
+ *
+ * The function is available if @kconfig{CONFIG_BT_DF_CONNECTION_CTE_TX} is enabled.
+ *
+ * @note @p params contents may be not completely validated and forwarded to Controller.
+ * If both BT_DF_CTE_TYPE_AOD_1US and BT_DF_CTE_TYPE_AOD_2US are not set in the bitfield,
+ * the bt_df_conn_cte_tx_param.num_ant_ids and bt_df_conn_cte_tx_param.ant_ids parameters
+ * will be ignored.
```

Comment:
I would skip the first line: "May not be completely validated" is a bit weird, and isn't very specific to the user.

I would also use `neither` instead of `and` + `not`, so it becomes something like
```suggestion
 * @note If neither BT_DF_CTE_TYPE_AOD_1US or BT_DF_CTE_TYPE_AOD_2US are set
 * in the bitfield, then the bt_df_conn_cte_tx_param.num_ant_ids and
 * bt_df_conn_cte_tx_param.ant_ids parameters will be ignored.
```

Reply:
Ok, I've just applied your suggestion. Thank you :-)